daterangepicker.js

```diff
@@ -603,9 +603,11 @@
             this.renderCalendar('right');
 
             //highlight any predefined range matching the current start and end dates
-            this.container.find('.ranges li').removeClass('active');
-            if (this.endDate == null) return;
-
+            if (this.endDate == null) {
+                return;
```

Comment:
Looks like two two rows are really just switched, but the commit confuses things by introducing the else-section, which makes it less obvious that `this.container.find()...` and `this.calculateChosenLabel()` go together.

Reply:
Maybe addClass/removeClass "active" should move into calculateChosenLabel()